src/main/java/com/thangle/api/book/BookController.java

```diff
@@ -24,4 +24,28 @@ public class BookController {
     public List<BookDTO> findAll() {
         return toBookDTOs(bookService.findAll());
     }
+
+    @Operation(summary = "Find a specific book by id")
+    @GetMapping("{id}")
+    public BookDTO findById(final @PathVariable UUID id) {
+        return toBookDTO(bookService.findById(id));
+    }
+
+    @Operation(summary = "Create a specific book")
+    @PostMapping
+    public BookDTO create(final @RequestBody Book book) {
+        return toBookDTO(bookService.create(book));
+    }
+
+    @Operation(summary = "Update a specific book")
+    @PutMapping("{id}")
+    public BookDTO update(final @PathVariable UUID id, final @RequestBody BookDTO bookDTO) {
```

Comment:
Please create another DTO for RequestBody since we won't use all of the fields in BookDTO (e.g createdAt, updatedAt, ...)